### PR TITLE
feat: delete "sentence" shortcut

### PR DESCRIPTION
**Pull request by Xazin**
Closes: #4

Need to check if the commands are properly setup for Linux and Windows!

### CODE REVIEW

File: lib/src/service/shortcut_event/built_in_shortcut_events.dart [hunk 1]
```diff
@@ -63,11 +63,18 @@ List<ShortcutEvent> builtInShortcutEvents = [
...
     linuxCommand: 'ctrl+backspace',
     handler: cursorLeftWordDelete,
   ),
+  ShortcutEvent(
+    key: 'Cursor sentence delete',
+    command: 'meta+backspace',
+    windowsCommand: 'alt+backspace',
+    linuxCommand: 'alt+backspace',
+    handler: cursorLeftSentenceDelete,
+  ),
```
Comment (Xazin) [inline lib/src/service/shortcut_event/built_in_shortcut_events.dart:77]: Need to verify that these commands are what we want.

They are different on MacOS than eg. Windows, due to the fact that they both have different platform behavior.

**Comment by LucasXu0** [comment]
Hey, @Xazin. Can you add more tests for the missing cover line?

**Comment by Xazin** [comment]
I will look at it tonight 👍

**Comment by LucasXu0** [comment]
There are some lines of code that don't have test coverage. I think it would be helpful to have test coverage for all of the lines. Would you be willing to work on that?

**Comment by Xazin** [comment]
> There are some lines of code that don't have test coverage. I think it would be helpful to have test coverage for all of the lines. Would you be willing to work on that?

Yes for sure. 

I ended up busy this weekend with another project. But I'm back looking at this, will make things a bit prettier and improve test coverage for all of `arrow_keys_handler.dart`.

File: test/service/internal_key_event_handlers/arrow_keys_handler_test.dart [hunk 2]
```diff
@@ -92,6 +154,71 @@ void main() async {
...
+    expect(
+      editor.documentSelection,
+      Selection.single(
+        path: [0],
+        startOffset: 19,
+      ),
+    );
+  });
+
+  testWidgets('Presses alt + arrow left key, move the cursor one left right',
```
Comment (Xazin) [inline test/service/internal_key_event_handlers/arrow_keys_handler_test.dart:193]:
```suggestion
  testWidgets('Presses alt + arrow left key, move the cursor one word left',
```

File: test/service/shortcut_event/shortcut_event_test.dart [hunk 3]
```diff
@@ -209,15 +234,56 @@ void main() async {
...
+          key: LogicalKeyboardKey.backspace,
+          isMetaPressed: true,
+        );
+      } else {
+        await editor.pressLogicKey(
+          key: LogicalKeyboardKey.backspace,
+          isControlPressed: true,
+          isAltPressed: true,
+        );
+      }
```
Comment (Xazin) [inline test/service/shortcut_event/shortcut_event_test.dart:275]:
```suggestion
      await editor.pressLogicKey(
        key: LogicalKeyboardKey.backspace,
        isMetaPressed: Platform.isMacOS,
        isControlPressed: !Platform.isMacOS,
        isAltPressed: !Platform.isMacOS,
      );
```We can do this everywhere else where applicable as well.

**Comment by LucasXu0** [comment]
LGTM.

**Pull request merged**